### DISCUSSION TITLE
Disappearing Request after Media is available (Sonarr/Jellyseerr)

### DISCUSSION THREAD

**Discussion by chaosfreak93** [seerr-team/seerr#1072] [Q&A]
Hello,

As soon as requests are approved and become available, the request is deleted from Jellyseer and the database and the user receives the request back, if there is a limit. Notifications about media availability are sent. 
It only affects Sonarr and therefore series.

Does anyone have any idea why this could be?

New Clean Jellyseerr 2.0.1 Docker Setup. Only manuelly transfered Watchlist from Old to New Setup.

**Comment by fallenbagel** [comment]
This should not happen. Could you hop onto discord to troubleshoot this? As we carry out support there. I might have an idea or two. Just create a #support thread in discord server and post logs too

**Comment by chaosfreak93** [comment]
Done, opened post in Discord
